cmd/sealos/cmd/root.go

```diff
@@ -28,12 +28,13 @@ import (
 var (
 	debug          bool
 	clusterRootDir string
+	runtimeRootDir string
 )
 
 // rootCmd represents the base command when called without any subcommands
 var rootCmd = &cobra.Command{
 	Use:   "sealos",
-	Short: "simplest way install kubernetes tools.",
+	Short: "simplest tool for installing kubernetes tools.",
```

Comment:
- Manage clusters lifecycle
   - Quickly install HA Kubernetes clusters
   - Add / remove nodes
   - Clean the cluster, backup and auto recovering, etc.
- Download and use OCI-compatible distributed applications
   - OpenEBS, MinIO, Ingress, PostgreSQL, MySQL, Redis, etc.
   - Customize your own distributed applications
   - Using Sealosfile to build distributed applications images, saving all dependencies.
   - Push distributed applications images to Docker Hub.
   - Combine multiple applications to build your own cloud platform.
- sealos cloud
   - Have a full public cloud capability, and run it anywhere

Reply:
`a unified and simplest tool to manage cloudnative toolset` 😄